repo2docker/app.py

```diff
@@ -176,6 +176,8 @@ def fetch(self, url, ref, checkout_path):
             sys.exit(1)
 
         if ref:
+            if len(ref.split('/')) != 2:
+                raise ValueError('--ref must be of the form remote/ref')
```

Comment:
This caused travis to fail.

I would change the error message to `"Expected --ref to be of the form remote/ref, but got %s" % ref` so that users can see what value arrived.